dashboard/app/assets/stylesheets/application.css.scss

```diff
@@ -30,6 +29,8 @@
  *= require details-polyfill
  */
 
+// TODO: Move into code-studio.scss when it exists.
+@import "/code-studio/assets/video-js/video-js.css";
```

Comment:
This compiles into a plain CSS @import directive:
```css
@import url(/code-studio/assets/video-js/video-js.css);
```
This works fine for now, although it adds a file for the client to download.  Eventually I'd like to concatenate this into a code-studio version of application.css, though it'll take some doing since sass doesn't have native support for concatenating in plain css files (which is pretty dumb, since they're valid scss).